### ISSUE TITLE
Feature Request: Repeat Creation to create multiple back to back

### ISSUE THREAD

**Issue by davids-tips** [elebumm/RedditVideoMakerBot#405]
Please add the functionality to create a custom amount back to back so it can be ground out.

**Comment by JasonLovesDoggo** [comment]
implemented in my [fork](https://github.com/JasonLovesDoggo/RedditVideoMakerBot)

**Comment by davids-tips** [comment]
> 

have you created a pull request?

**Comment by davids-tips** [comment]
@JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?

**Comment by JasonLovesDoggo** [comment]
> @JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?
> 

I think I'm going to change the way it does it right now but what it does is it saves all the done videos in a file called videos.json and before it picks the thread it checks if it's already been done

**Comment by davids-tips** [comment]
> > @JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?
> 
> I think I'm going to change the way it does it right now but what it does is it saves all the done videos in a file called videos.json and before it picks the thread it checks if it's already been done

thats what i was trying to add to my edit XD

**Comment by JasonLovesDoggo** [comment]
> > > @JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?
> > 
> > I think I'm going to change the way it does it right now but what it does is it saves all the done videos in a file called videos.json and before it picks the thread it checks if it's already been done
> 
> thats what i was trying to add to my edit XD
> 

I wish there was a way I could make multiple forks of a single repository

**Comment by davids-tips** [comment]
> > > > @JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?
> > > 
> > > 
> > > I think I'm going to change the way it does it right now but what it does is it saves all the done videos in a file called videos.json and before it picks the thread it checks if it's already been done
> > 
> > 
> > thats what i was trying to add to my edit XD
> 
> I wish there was a way I could make multiple forks of a single repository

If you are trying to update the base code just download the .zip and create another branch based off the master one in your fork

**Comment by JasonLovesDoggo** [comment]
> > > > > @JasonLovesDoggo how does your fork prevent duplicates? does it create a file that has names in them or does it look at the files in the "results" folder?
> > > > 
> > > > 
> > > > I think I'm going to change the way it does it right now but what it does is it saves all the done videos in a file called videos.json and before it picks the thread it checks if it's already been done
> > > 
> > > 
> > > thats what i was trying to add to my edit XD
> > 
> > I wish there was a way I could make multiple forks of a single repository
> 
> If you are trying to update the base code just download the .zip and create another branch based off the master one in your fork

Good idea

**Comment by davids-tips** [comment]
also @JasonLovesDoggo is the 29 vid limit a arbrtrary number or is it enforecd in the code?

**Comment by JasonLovesDoggo** [comment]
> also @JasonLovesDoggo is the 29 vid limit a arbrtrary number or is it enforecd in the code?
> 

It's not enforced
As long as it's under the 64 bit integer limit it will work

**Comment by davids-tips** [comment]
@JasonLovesDoggo is there a way that i can have it ignore the error so it can continue onto the next one if the video errors out?

**Comment by JasonLovesDoggo** [comment]
> @JasonLovesDoggo is there a way that i can have it ignore the error so it can continue onto the next one if the video errors out?
> 

Wrap the entire thing in a try statement

**Comment by William9923** [comment]
> @JasonLovesDoggo is there a way that i can have it ignore the error so it can continue onto the next one if the video errors out?

Tldr: If error happens, could we skip it and move on to the creation of the next video.
Correct me if im wrong

**Comment by JasonLovesDoggo** [comment]
either way IMPLEMENTED
